components/suggestion/at_mention_provider/at_mention_provider.jsx

```diff
@@ -59,13 +59,16 @@ export default class AtMentionProvider extends Provider {
             profileSuggestions.push(...usernameSuggestions);
         }
         if (profile.first_name) {
-            profileSuggestions.push(profile.first_name.toLowerCase());
+            const firstNameSuggestions = getSuggestionsSplitBy(profile.first_name.toLowerCase(), ' ');
+            profileSuggestions.push(...firstNameSuggestions);
         }
         if (profile.last_name) {
-            profileSuggestions.push(profile.last_name.toLowerCase());
+            const lastNameSuggestions = getSuggestionsSplitBy(profile.last_name.toLowerCase(), ' ');
+            profileSuggestions.push(...lastNameSuggestions);
         }
         if (profile.nickname) {
-            profileSuggestions.push(profile.nickname.toLowerCase());
+            const nicknameSuggestions = getSuggestionsSplitBy(profile.nickname.toLowerCase(), ' ');
+            profileSuggestions.push(...nicknameSuggestions);
         }
```

Comment:
These 3 conditionals could be done in a loop with `['first_name', 'last_name', 'nickname'].forEach`.